styles/components/_misc.scss

```diff
@@ -95,6 +95,10 @@
   }
 }
 
+#scamWarning {
```

Comment:
To be theme friendly, instead of hard coding this color, just add the `clrTErr` class to the anchor.